main.py

```
@@ -0,0 +1,179 @@
+                    documents.append(f"Documento sin contenido claramente extraíble: {result.document.name if result.document else 'N/A'}")
+            else:
+                documents.append("Resultado de búsqueda sin documento.")
+
+
+        if not documents:
+            document_content = "No se encontraron documentos relevantes en Vertex AI Search."
+        else:
+            # Concatena los documentos encontrados para usarlos como contexto
+            document_content = "\n\n".join(documents)
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)